Core/PostProcessingHelper.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Core;
+
+use Wirecard\Oxid\Model\PaymentMethod;
+use Wirecard\Oxid\Model\RatepayInvoicePaymentMethod;
+use Wirecard\Oxid\Model\Transaction;
+
+/**
+ * Class PostProcessingHelper
+ *
+ * @since 1.2.0
+ */
+class PostProcessingHelper
+{
+
+    /**
+     * Returns whether to use order items or amount in the panel
+     *
+     * @param Transaction $oTransaction
+     *
+     * @return bool
+     *
+     * @since 1.2.0
+     */
+    public static function shouldUseOrderItems($oTransaction)
```

Comment:
Much better to solve it over the PaymentMethod classes, so no upkeep of this method is needed.

Reply:
see https://github.com/wirecard/oxid-ee/pull/128#discussion_r293279971